timerit/core.py

```diff
@@ -416,21 +427,28 @@ def call(self, func, *args, **kwargs):
 
     def __iter__(self):
         if self.verbose >= 3:
-            print(self._status_line(tense='present'))
+            print(self._status_line())
 
         self.n_loops = 0
         self.total_time = 0
 
-        # TODO: if num is unspecified, can we just run for a set amount of time
-        # and then stop?
-
         bg_timer = self._bg_timer
         fg_timer = self._fg_timer
+
         # disable the garbage collector while timing
         with _SetGCState(enable=False):
             # Core timing loop
-            for _ in it.repeat(None, self.num):
```

Comment:
I remember choosing `it.repeat` here as it was the fastest way to loop for a fixed number of iterations. But looking at the implementation it seems like it probably doesn't matter because nothing is timed while iterations are being checked.

So this is probably fine, I'm just commenting as a sanity check for myself.